zuul.d/adoption.yaml

```diff
@@ -200,7 +200,7 @@
     name: cifmw-adoption-base-source-multinode
     parent: base-extracted-crc
     abstract: true
-    timeout: 14400
+    timeout: 18000
```

Comment:
How this relevant for OSPCIX-758? the issue was specific to standalone jobs but this is change base multinode jobs, periodic multinodes already have 18000 timeout set and in check we not running the multinodes ones

Reply:
you are right, it is not related to standalone jobs in that CIX, so we don't need this change